Blog post for the Prometheus OpenCost Exporter

mattray opened the pull request:
New integration adding it to the blog, docs to follow

dwbrown2 commented:
Dropped comments on work doc draft!

dwbrown2 commented:
Like the screenshot addition, but should you show a metric actually tried to cost (e.g. node_hourly_cost) vs resource allocation? That seems more relevant. 

Also, I could go either way on Prom logo... don't feel it's required.

mattray commented:
> Like the screenshot addition, but should you show a metric actually tried to cost (e.g. node_hourly_cost) vs resource allocation? That seems more relevant.

Updated to `node_cpu_hourly_cost` against a GKE node 

> 
> Also, I could go either way on Prom logo... don't feel it's required.

Removed

dwbrown2 commented:
Looking good to me. Maybe worth getting one more set of eyes on it? 

When are you hoping to launch?

jessegoodier commented on blog/2023-12-18-prometheus-opencost-exporter/index.mdx:
````diff
@@ -0,0 +1,29 @@
+helm install [RELEASE_NAME] prometheus-community/prometheus-opencost-exporter
+```
+
+And this will install the latest version of the Prometheus OpenCost Exporter (currently 1.107.1). The Prometheus OpenCost Exporter will be kept in sync with future releases of OpenCost going forward. If you want to change any of the default settings, you can provide your own [values.yaml](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-opencost-exporter/values.yaml ...
````
The version (1.107.1) in docs tend to be forgotten and then the doc looks stale. Your call if you want to maintain this?

mattray replied:
https://github.com/prometheus-community/helm-charts/pull/4097 it's just going to be part of the OpenCost release process to keep this synced
